### DISCUSSION TITLE
CheckIt Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#43] [Show and tell]
Lets use this space to discuss any needed improvements to the Checkit exercises now available at https://teambasedinquirylearning.github.io/precalculus/2024/exercises/#/bank/

**Comment by AbbyANoble** [comment]
PR6: Asymptotes has an extra s in Tasks 3 and 4!

**Reply by siwelwerd** [reply]
Went ahead and made a PR to fix this! See #51

**Reply by AbbyANoble** [reply]
"unfortunate typo" 😂

**Reply by StevenClontz** [reply]
I... assume it was okay to review and merge and not wait on @TienChih's review here. :-)

**Reply by siwelwerd** [reply]
I thought he might see it before you with you being in sunny Pasadena this week :)

**Reply by StevenClontz** [reply]
The clickbait PR title caught me lol

**Comment by AbbyANoble** [comment]
PR4 - The 1st version is hard for me to really see what's happening at x=-3 and x=-2. Maybe it's my old eyes, but I feel like I need to zoom in. Especially compared to versions 3 & 4 where I can really tell whether the graph crosses or "bumps" the axis at the zeros. Is there a way to make sure it's more obvious what is going on?

<img width="701" alt="Screenshot 2023-12-08 at 11 43 42 AM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/85640990/29b1899c-c320-4d02-9339-aefc91295482">

**Comment by AbbyANoble** [comment]
Also in PR4 - multiplicity is misspelled each time in task 2.
